.github/workflows/tests-runtime-arm.yaml

```diff
@@ -0,0 +1,70 @@
+name: Unit Tests Arm
```

Comment:
AFAICS, this workflow runs the same tests as `.github/workflows/integration-test.yaml`. Would it make sense to integrate the two and use a matrix for the `runs-on` (and make the docker installation dependent on `ubuntu-22.04-arm64`.

Reply:
Yeah, I totally miss there was a file already for this. I'll adapt it.